pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -499,10 +499,11 @@ func (s *usmHTTP2Suite) TestRawTraffic() {
 	}
 
 	tests := []struct {
-		name              string
-		skip              bool
-		messageBuilder    func() [][]byte
-		expectedEndpoints map[usmhttp.Key]int
+		name                      string
+		skip                      bool
+		validateTCPSplitTelemetry bool
```

Comment:
this flag is used only once for a single subtest
I think it is better to move it out to a different test